mmrazor/engine/runner/evolution_search_loop.py

```diff
@@ -99,7 +105,7 @@ def __init__(self,
                 correct init candidates file'
 
         self.top_k_candidates = Candidates()
-        if resource_estimator_cfg is None:
+        if resource_estimator_cfg:
```

Comment:
Should it be `if resource_estimator_cfg is None`?

Reply:
default resource_estimator_cfg is dict.

Reply:
Then directly use `self.estimator = ResourceEstimator(**resource_estimator_cfg)`